addons/message_update_v2.13/manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+  "api_version": "0.1",
+  "id": "message_update_v2.13",
+  "name": "Update to Mozilla VPN 2.13",
+  "type": "message",
+  "conditions": {
+    "max_client_version": "2.12.9"
+  },
+  "javascript": {
+    "enable": "enable.js"
+  },
+  "message": {
+    "date": 1671066000,
```

Comment:
do we need to add the setDate js function so we don't have a hardcoded timestamp?

Reply:
Ahhh, I see it is in enable.js. nevermind